torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -111,6 +111,7 @@ def is_copy_node(node, modules):
         torch.flatten,
         torch.mean,
         operator.floordiv,
+        torch.channel_shuffle,
```

Comment:
do we need to include torch.functional.channel_shuffle as well? and why this is not catched by the test?

Reply:
Looks like `torch.channel_shuffle` and `torch.nn.functional.channel_shuffle` are actually the same thing:
```
>>> import torch
>>> print(torch.channel_shuffle)
<built-in method channel_shuffle of type object at 0x7f74ab048560>
>>> print(torch.nn.functional.channel_shuffle)
<built-in method channel_shuffle of type object at 0x7f74ab048560>
>>> print(torch.nn.ChannelShuffle)
<class 'torch.nn.modules.channelshuffle.ChannelShuffle'>
```

Reply:
BTW, do you know where are `torch.channel_shuffle`/`torch.nn.functional.channel_shuffle` defined? I cannot find implementation in source code, only got this:
https://github.com/pytorch/pytorch/blob/09965957cd8ecc696852e73022892b3ad4475783/torch/nn/functional.py#L3576

Reply:
this is where F.channel_shuffle is defined I think, it just says F.channel_shuffle is the same as torch.channel_shuffle

Reply:
maybe we can add a test saying `torch.nn.functional.channel_shuffle` is the same as `torch.channel_shuffle` so we only need to support lowering for `torch.channel_shuffle`?

Reply:
Do you mean adding a line of comment or an assertion here? Or a separate test that only has one line of assertion?

Reply:
> Do you mean adding a line of comment or an assertion here? Or a separate test that only has one line of assertion?

Hi @jerryzh168 Please take a look

Reply:
I think you can add a comment

Reply:
Added